config/database.yml.sample

```diff
@@ -2,6 +2,7 @@ default: &default
   adapter: postgresql
   encoding: utf8
   username: postgres
+  prepared_statements: false
```

Comment:
maybe source this from ENV so we can manually update the deployment in k8s if needed?